packages/babel-plugin-fbt/FbtUtil.js

```diff
@@ -116,6 +116,11 @@ function checkOption(option, validOptions, value) {
   return option;
 }
 
+function isBoolAttribute({name}){
+    return name === 'doNotExtract' || name === 'number'
```

Comment:
Let's implement this as a static map declared outside the function instead.
Something like the following
```
const keyMirror = require('fbjs/lib/keyMirror');

// ...

const SHORT_BOOL_CANDIDATES = keyMirror({
  doNotExtract: null,
  number: null,
});

function canBeShortBoolAttr(name) { 
  return name in SHORT_BOOL_CANDIDATES;
}
```